util/config/config.go

```diff
@@ -177,6 +178,10 @@ var (
 		LogPath:                      "Log",
 		ChainDBPath:                  "ChainDB",
 		WalletFile:                   "wallet.json",
+		HttpWssCert:                  "cert/certificate.pem",
+		HttpWssKey:                   "cert/key.pem",
+		HttpJsonCert:                 "cert/certificate.pem",
```

Comment:
Two minor naming suggestions:
1. Either use HttpWsCert/HttpJsonCert, or HttpWssCert/HttpsJsonCert for naming consistency
2. Typically people use certs/ instead of cert/

Reply:
Fixed.